### PR TITLE
Add postgis  to conda-forge

### PR DESCRIPTION
**Pull request by sodre**

### CODE REVIEW

**Comment by sodre** [comment]
@ocefpaf,

This is ready for review as well. I am skipping the windows builds for now.

Alternative option to postgresql-contrib:
   - Ensure the contrib directory is being built in the original postgresql-feedstock. I am happy to open a PR for that if you prefer.

**Comment by mariusvniekerk** [comment]
Happy about helping out as a maintainer

File: recipes/postgis/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    --with-xml2config=$PREFIX/bin/xml2-config \
+    --with-projdir=$PREFIX \
+    --with-libiconv=$PREFIX \
+    --with-jsondir=$PREFIX \
+    --with-pcredir=$PREFIX \
+    --with-gettext \
+    --with-raster \
+    --with-topology
+make
+make install
```
Comment (ocefpaf) [inline recipes/postgis/build.sh]: Is there a `make check` we can run?

File: recipes/postgis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [ win ]
+
+requirements:
+  build:
+    - gdal
+    - geos
```
Comment (ocefpaf) [inline recipes/postgis/meta.yaml]: You have to pin this according to https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L64-L130

**Comment by bgruening** [comment]
> Happy about helping out as a maintainer
me too

File: recipes/postgis/build.sh (hunk 1 above)
Comment (sodre) [reply recipes/postgis/build.sh]: I am working on getting make check to work. The builds will be in a failure state until that is done.

**Comment by sodre** [comment]
I will wait until conda-forge/postgresql-feedstock#19 is merged to confirm the build error on OSX.

**Comment by sodre** [comment]
@ocefpaf,

The PostGIS library is now ready to merge. Make check is running completely fine on Linux. On the Mac it is currently disabled because of some issue with shp2pgsql during build time. However, I still execute the 3 binaries (shp2pgsql, pgsql2shp and raster2pgsql) on both Mac and Linux.

**Comment by bgruening** [comment]
Super cool @sodre!

**Comment by ocefpaf** [comment]
Sorry guys, out for a conference and slow to process all the traffic here in conda-forge. Thanks @sodre for the PR and @bgruening for the merge. I'll check if there is anything I can add to this later but it looks great.

**Pull request merged**